.github/workflows/ci.yaml

```diff
@@ -0,0 +1,78 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - _integration/**/*
+      - test_consume_workiva/**
+    tags:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dart-lang/setup-dart@v1
+        with:
+          sdk: 2.19.6
+
+      - name: Print Dart SDK version
+        run: dart --version
+
+      - name: Install dependencies
+        run: dart pub get
+
+      - name: Validate code
+        run: dart run dependency_validator
+
+      - name: Check format
+        run: dart run dart_dev format --check
+
+      - name: Analyze code
+        run: dart analyze
```

Comment:
might be best to stay consistent with how we are running dart commands, thoughts on updating this to use ddev?

```
dart run dart_dev analyze
```